src/resources/packages/wizard/components/tabs/organizer/tab.tsx

```diff
@@ -126,7 +126,7 @@ const OrganizerContent = ({moveToNextTab, skipToNextTab}) => {
 			return true;
 		}
 
-		const phonePattern = /^\+?\d?[\s.-]?(?:\(\d{3}\)|\d{3})[\s.-]?\d{3}[\s.-]?\d{4}$/;
+		const phonePattern = /^(?:\+?\d?[\s.-]?(?:\(\d{3}\)|\d{3})[\s.-]?\d{3}[\s.-]?\d{4}|)$/;
```

Comment:
1. no need to create a group here
2. same issue as above - rather than alter the regex, add a condition that validates an empty string before we get here.